src/pages/review.tsx

```diff
@@ -0,0 +1,138 @@
+import styled from "@emotion/styled";
+import type { NextPage } from "next";
+import Head from "next/head";
+import Image from "next/image";
+import StarFilled from "../../public/assets/star_filled.png";
+import StarOutlined from "../../public/assets/star_outlined.png";
+
+const ReviewPage: NextPage = () => {
+  return (
+    <div>
+      <Head>
+        <title>리뷰 작성</title>
+        <meta name="description" content="Generated by create next app" />
+        <link rel="icon" href="/favicon.ico" />
+      </Head>
+      <Cont>
+        <Title>마이 페이지</Title>
+        <ReviewCont>
+          <ReviewTitle>리뷰작성</ReviewTitle>
+          <Content>
+            <ImgContent>
+              <ImgCont></ImgCont>
+              <AddImgBtn>이미지 추가</AddImgBtn>
+            </ImgContent>
+            <UserReviewCont>
+              <Text>별점 및 리뷰</Text>
+              <StarCont>
+                <Image src={StarFilled} alt="별점" width={22} height={22} />
+                <Image src={StarFilled} alt="별점" width={22} height={22} />
+                <Image src={StarFilled} alt="별점" width={22} height={22} />
+                <Image src={StarOutlined} alt="별점" width={22} height={22} />
+                <Image src={StarOutlined} alt="별점" width={22} height={22} />
+              </StarCont>
+              <ReviewInp
+                rows={8}
+                cols={80}
+                placeholder="상품에 대한 솔직한 리뷰를 작성해주세요."
+              ></ReviewInp>
+              <ReviewSubmitBtn type="submit">등록하기</ReviewSubmitBtn>
```

Comment:
p4: 이름이 겹치지만 않는다면 이렇게 해줘도 좋을 것 같아요
```suggestion
              <ReviewSubmit type="submit">등록하기</ReviewSubmit>
```